doc/cheatsheet.rst

```diff
@@ -287,3 +287,50 @@ and normal for :math:`\nu = \infty`.
     - \frac{1}{\mu_i}\Bigg\}{\mu_i'}x_{ij} + \lambda (1 - \alpha) \beta_j
 
 where :math:`\mu_i' = \frac{1}{1 + \exp(-z_i)}`.
+
+
+Negative Binomial
+-------------------------------
+**Mean Function**
+
+.. math::
+
+    z_i = \beta_0 + \sum_j \beta_j x_{ij} \\
+    \mu_i = \log(1 + \exp(z_i))
+
+**Log-likelihood function**
```

Comment:
@geektoni 

i have not checked the derivations by hand yet, but do note that the [pmf and log likelihood specified in wikipedia](https://en.wikipedia.org/wiki/Negative_binomial_distribution#Maximum_likelihood_estimation) defines `r` (which is `\theta` in your cheatsheet) as a number of failures whereas `np.random.negative_binomial(n, p)` defines `n` as number of successes.

It would be great to carefully check the log-likelihood formula (and therefore the loss, grad, hess formulas) to make sure they are consistent with `numpy`.

Reply:
Yes, I think I will need to check more closely the `numpy` negative binomial implementation because, as you pointed out, it seems like it is not consistent with my derivations.